lively.lang/package.json

```diff
@@ -32,7 +32,9 @@
       }
     }
   },
-  "dependencies": {},
+  "dependencies": {
+    "articles": "^0.2.2"
```

Comment:
Is this package in ES6 module syntax? If not, this will cause lively bundles to fail, since our bundler only supports ESM format.

Reply:
I don't think it is: https://jspm.dev/npm:articles@0.2.2!cjs. How would I check this? install.sh ran without problem

Reply:
Well usually by inspecting the source files that are pulled by npm. Since npm performs a bunch of preprocessing of scripts, there is no easy way of telling aside from directly inspecting the pulled source files.
Alternatively you can just fetch the package via jspm, then you are on the safe side since jspm transpiles everything tp esm like magic.

Reply:
In the node_modules folder the only code file starts with `(function() {` and ends with  `}).call(this); ` so I don't think it is an ES module. So should I add the dependency as a link on top of lively.lang?

Reply:
Yes, but putting the paths inside the dependencies wont work, you need to add a mapping like so: 
```
"systemjs": {
    "map": {
      "articles": "https://jspm.dev/npm:articles@0.2.2"
    }
  },

```